Encountered an error synchronizing team <%= view.link_to_team %> with LDAP -
<% case view.entry.hit["error"] %>
<% when :missing_entry %>the entry <span class="context"><%= view.entry.hit["dn"] %></span> was not found.
<% else %>
  <span class="context"><%= view.entry.hit["error"] %>.</span>
<% end %>
